dask_sql/physical/rel/convert.py

```diff
@@ -68,9 +68,7 @@ def convert(cls, rel: "LogicalPlan", context: "dask_sql.Context") -> dd.DataFram
                     f"'{node_type}' is a relational algebra operation which doesn't require a direct Dask task. \
                     Omitting it from the resulting Dask task graph."
                 )
-                return context.schema[rel.getCurrentNodeSchemaName()].tables[
-                    rel.getCurrentNodeTableName()
-                ]
+                return BaseRelPlugin.assert_inputs(rel, 1, context)[0]
```

Comment:
contemplating if this should become it's own operation instead of a special case here